### ISSUE TITLE
Vue tag auto-close feature

### ISSUE THREAD

**Issue by ElectricS01** [zed-industries/zed#13504]
### Check for existing issues

- [X] Completed

### Describe the feature

Add tag auto-close to Vue like in https://github.com/zed-industries/zed/issues/5267

Currently when modifying a tag in `.vue` files you must modify both the opening and closing tag manually.

It would be really helpful for Vue users if tag auto-close could be added in the same way it was done for the HTML language server.

### If applicable, add mockups / screenshots to help present your vision of the feature

none

**Comment by markiol** [comment]
+1, It's really annoying, that it's not implemented yet!

**Comment by realcuesn** [comment]
+420 really need an implentation for this

**Comment by s314cy** [comment]
this lacking feature is the _only_ reason why I haven't switched to zed yet

**Comment by reslear** [comment]
For a layout dev this is such an important function that we have to back to vscode :(

**Comment by probably-neb** [comment]
This is now supported as of the latest release! (as of #25681). If you update Zed (as well as the Vue extension) to the latest version tag auto-closing will be enabled by default. Cheers!

**Comment by acuteaura** [comment]
This breaks the common default [vue/html-self-closing](https://eslint.vuejs.org/rules/html-self-closing) default for void tags like `br`.

**Comment by acuteaura** [comment]
It also ignores `jsx_tag_auto_close.enabled` in the settings... @probably-neb

**Comment by probably-neb** [comment]
Yes, it does not try to be smart and determine which tags you'd like to be self closing. You can avoid having a self closing tag be auto-closed by typing `/>` at the end of the tag instead of `>`, as it won't attempt to auto-close an already self-closed tag.

On the ignoring `jsx_tag_auto_close.enabled` setting getting ignored, could you please create a new issue @acuteaura and include your settings file and reproduction steps? Thanks
